Patch for upstream: fixes issue that tumble and duplex are set by default.

diff -r b0ed3b2021c3 -r 5c5f3a031e0e src/paps.c
--- a/src/paps.c	Mon Feb 15 16:21:50 2016 +0900
+++ b/src/paps.c	Mon Feb 15 16:21:51 2016 +0900
@@ -695,16 +695,16 @@
       page_width = page_height;
       page_height = tmp;
       if (do_tumble < 0)
-        do_tumble = TRUE;
+        do_tumble = FALSE;
       if (do_duplex < 0)
-        do_duplex = TRUE;
+        do_duplex = FALSE;
     }
   else
     {
       if (do_tumble < 0)
-        do_tumble = TRUE;
+        do_tumble = FALSE;
       if (do_duplex < 0)
-        do_duplex = TRUE;
+        do_duplex = FALSE;
     }
   
   page_layout.page_width = page_width;
